src/main/java/quotes/App.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package quotes;
+
+import com.google.gson.Gson;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
```

Comment:
Your code looks great Quang, and like how you are commenting what you are doing in the methods. 👍